specs/subresourceintegrity/spec.markdown

```diff
@@ -746,7 +746,10 @@ Much of the content here is inspired heavily by Gervase
 Markham's [Link Fingerprints][] concept, as well as WHATWG's [Link Hashes][].
 
 A special thanks to Mike West of Google, Inc. for his invaluable contributions
-to the initial version of this spec.
+to the initial version of this spec. Additonally, (in no particular order) 
+Eduardo Vela, Michal Zalewski, Mark Nottingham, Anne Van Kesteren, 
```

Comment:
If it's going to be "in no particular order", we should at least alphabetize, IMO.